js/components/profile.js

```diff
@@ -30,15 +35,35 @@ class Profile extends LitElement {
 
   async firstUpdated() {
     this.user = await userRepository.getByName(this.location.params.name);
+    this.authUser = fireauth.currentUser;
+    this.followed = userRepository.isFollowing(this.user.id);
+    this.follow = () => {
```

Comment:
`this.follow` and `this.follow` sont pareils, peut-être juste faire `this.toggleFollow` et en fonction de `this.followed` tu add ou del